test/lib/ansible_test/_data/requirements/sanity.rstcheck.txt

```diff
@@ -1 +1 @@
-rstcheck
+rstcheck >=3.3.1  # required for sphinx version > 1.8
```

Comment:
```suggestion
rstcheck >=3.3.1  # required for sphinx version >= 1.8
```

Reply:
added to the test fix - i wasn't supposed to put  this in the rstcheck file ;-)